supportedBackends/spark/src/main/scala/cromwell/backend/impl/spark/SparkRuntimeAttributes.scala

```diff
@@ -0,0 +1,82 @@
+package cromwell.backend.impl.spark
+
+import cromwell.backend.MemorySize
+import cromwell.backend.validation.RuntimeAttributesDefault._
+import cromwell.backend.validation.RuntimeAttributesKeys._
+import cromwell.backend.validation.RuntimeAttributesValidation._
+import cromwell.core._
+import lenthall.exception.MessageAggregation
+import wdl4s.types.{WdlBooleanType, WdlIntegerType, WdlStringType, WdlType}
+import wdl4s.values.{WdlBoolean, WdlInteger, WdlString, WdlValue}
+
+import scalaz.Scalaz._
+import scalaz._
+
+object SparkRuntimeAttributes {
```

Comment:
Another note re the latest develop: We've centralised the RuntimeAttribute evaluation, so all you need to do is override the `RuntimeAttributeDefinitions` on the backend factory, and hey-presto, the engine will deliver your requested runtime attributes as part of the `jobDescriptor`